R/length-params.R

```diff
@@ -42,30 +38,28 @@ get_catch_types <- function(log_threshold = logger::DEBUG) {
       name = dplyr::case_when(
         name == "Thunnini" ~ "Tuna",
         name == "Selachimorpha" ~ "Shark",
+        name == "Reptantia" ~ "Nephropidae",
```

Comment:
Pending check with Alex to see whether most of the lobsters caught in East Timor are spiny lobsters or actual lobsters

Reply:
Just updated the LOX group with _Panulirus_ species

Reply:
Perfect! Just as with the other manual hacks it might be good to add a comment to the code. I find comments specially useful to explain *why* a piece of code unexpected code has been written.

Reply:
Yeah you are completely right, sorry for that!